equinor/EchoFramework
# chore: Security updates

## Prerequisites

Please make sure you can check the following boxes:


## Type(s) of Changes

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:


### Description

**Regular security updates:**
- updated echo packages
- updated other outdated packages, except for react router dom: that has a mayor updated, needs a refactor to make it work

**Additional changes:**
New version of EDS introduced some breaking changes in snackbar:
- updated affected code - toaster
It still displays properly, in the bottom left corner
![image](https://user-images.githubusercontent.com/13872908/142180032-7b53c802-7bd8-441e-8913-5010add47a0f.png)

- Added globalStyles: we have an issue with EDS tooltip, as it's z-index has fixed value of 250. Previously the coreLeft and right panels z-index was set to 500, so there the tooltips are behind the panels.
Added this global style to override EDS's z-index value.

Outdated dependencies before the update
![framework versions](https://user-images.githubusercontent.com/13872908/142180134-4d394aa5-bd8d-44f3-bed8-d3477c12d27d.png)


After
![image](https://user-images.githubusercontent.com/13872908/142180219-4aa4ab0b-33db-4f81-8e08-7f8386241943.png)

Vulnerabilities:
![image](https://user-images.githubusercontent.com/13872908/142180301-2f494d90-671a-4a12-8b0f-03d5f3a87d6c.png)

### How to test?
Build echoFramework: it should build without any issue.

Link echoFramework with Yalc to echopediaWeb.
Click around the app, see if anything is broken.
